Jenkinsfile

```diff
@@ -1,21 +1,22 @@
 #!groovy
 
-def runTest() {
+def runTest(env) {
     node {
         stage("checkout") {
             checkout scm
         }
         stage("test") {
             docker.build("codeception").inside {
-                sh "codecept run"
+                sh "./codecept.phar build"
+                sh "./codecept.phar run $env"
             }
         }
     }
 }
 
 if (env.APPIUM_ENDPOINT.contains("staging.testobject.org")) {
     lock (resource: params.TESTOBJECT_DEVICE) {
-        runTest()
+        runTest(' --env staging')
```

Comment:
Rather than needing to call `runTest(' --env staging')`, could you make it so we can call `runTest('staging')`. You could add the `--env` flag in the `runTest` function.